v2/src/plugins/markdownVariables.json

```diff
@@ -71,6 +86,28 @@
         "goEPModelsImport": "",
         "goSignIn": "SignIn",
         "pythonSignIn": "sign_in",
-        "nodeSignIn": "signIn"
+        "nodeSignIn": "signIn",
+        "nodeRecipeInitDefault": "",
+        "goRecipeInitDefault": "",
+        "pythonDefaultCallbackDefs": "",
+        "pythonRecipeInitDefault": "",
+        "reactRecipeInitDefault": ""
+    },
+    "thirdpartypasswordless": {
+        "getUserByEmailNode": "getUsersByEmail",
+        "getUserByEmailGo": "GetUsersByEmail",
+        "getUserByEmailPython": "get_users_by_email",
+        "docsLinkRecipeName": "thirdpartypasswordless",
+        "codeImportRecipeName": "thirdpartypasswordless",
+        "rid": "thirdpartypasswordless",
+        "recipeNameCapitalLetters": "ThirdPartyPasswordless",
+        "goModelName": "tpplessmodels",
+        "goGetUserById": "GetUserById",
+        "goEPModelsImport": "",
+        "reactRecipeInitDefault": "contactMethod: \"EMAIL\", // This example will work with any contactMethod\n",
+        "nodeRecipeInitDefault": "contactMethod: \"EMAIL\", // This example will work with any contactMethod\n        createAndSendCustomEmail: async (input) => console.log(input),\n        flowType: \"USER_INPUT_CODE_AND_MAGIC_LINK\", // This example will work with any flow.",
```

Comment:
Try and avoid these kinds of very specific variables. If this is needed, please check the indentation (it's not correct)